### ISSUE TITLE
BUG: Created Events Homepage and Manage Events page not current view

### ISSUE THREAD

**Issue by SELord** [SELord/Schedule-it#84]
When the user clicks on the Created Events button on Schedult-it Home or on the Manage Events page, they are shown a list view of events they have created. 

The issue is that the view defaults to show the week of the event that is the furthest in the future. 

The expected behavior is that the page defaults to show the current week view so the user sees which events are coming up next.

**Comment by SELord** [comment]
Fix coming in branch steve-week-9

For homepage bug, issue was in view_history.js where defaultDate was set to the last date in the list of all events created, which would set it to be the event furthest in the future. Fixed by copying the top 6 lines of code from event.js :
```//Get today's date 
let today = new Date();
let dd = String(today.getDate()).padStart(2, '0');
let mm = String(today.getMonth() + 1).padStart(2, "0");
let yyyy = today.getFullYear();
today = yyyy + '-' + mm + '-' + dd;
```
and setting defaultDate to today within the calendar object creation function within createdEventHist().

**Comment by SELord** [comment]
Same issue as in the Manage Events main screen. The problematic code in manage_events_main.js is as follows:
```
let mostRecent;
    if (pastEvents.length > 0) {
        mostRecent = pastEvents[pastEvents.length - 1]['start']; // fullcalendar.io can handle getting the date from a string with the date and time
    }
```
Then defaultDate is set to mostRecent. 

Code fixed in same way as view_history.js

**Comment by SELord** [comment]
Fixed with commit 54be69e and 1d4db43b3701b1971800ee057afc92798986c637
